frontend/src/lib/components/Sharing/SharingModal.tsx

```diff
@@ -38,6 +38,10 @@ export interface SharingModalBaseProps {
     title?: string
     previewIframe?: boolean
     additionalParams?: Record<string, any>
+    /**
+     * When generating a link to a recording, this form can be used to allow the user to specify a timestamp
+     */
+    recordingLinkTimeForm?: ReactNode
```

Comment:
this lets us put the timestamp selector next to the link button so it mirrors the other forms